src/main/views/courts/tabs/typesContent.njk

```diff
@@ -106,24 +106,183 @@
 
 {% endfor %}
 
-
 {{ govukCheckboxes({
   idPrefix: "court_types",
   name: "types",
-
-
   items: items
 }) }}
 
+{% if isSuperAdmin %}
+<hr/>
+<br/>
```

Comment:
We should prob avoid using bare bones <hr's and br's, and instead use div's with the gov uk classes instead

Reply:
replaced with  <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">